.github/workflows/test.yml

```
@@ -64,7 +62,11 @@ jobs:
         env:
           PGPASSWORD: pgroonga
         run: |
-          tag="${LATEST_VERSION}-${{ matrix.id }}"
+          if [ ${{ github.event_name }} = "schedule" ]; then
```

We should use environment variable as much possible to reduce evaluation timing:

```suggestion
          if [ ${GITHUB_EVENT_NAME} = "schedule" ]; then
```